### ISSUE TITLE
Recommend Lines: prediction of opposing team

### ISSUE THREAD

**Issue by ldacosta** [stradigilabs/NHL_stats#30]
When we recommend a line (circa https://github.com/stradigilabs/NHL_stats/blob/master/ReinforcementLearning/NHL/lines/recommender.py#L18 ) we assume that we will get as input the 4 lines that the opposing coach will send (this prediction is currently done by observing the historical choices made by the coaches). 
This strategy leave us open to any changes that could happen during the game - in particular if the opposing coach chooses to try a combination he tried in the past, but wasn't very popular, we would have no data to give a recommendation.
Proposed change: to include in the signature not only the 4 lines we expect but rather all lines that are somehow popular with the opposing coach.

**Comment by YounesZ** [comment]
Yeah, maybe we could give coaches the best lines to send against other less expected lines so they can adapt during games. However, I think this should constitute a side note to the lines recommendation report to avoid providing overwhelming information
